src/modules/journal/journal.selector.ts

```diff
@@ -39,7 +51,7 @@ export const canNavigateToNextDay = (journal: JournalModel): boolean => {
     traverseWeekend = true;
   }
 
-  return nextDayAsDate < dayAfterTomorrowAsDate || traverseWeekend;
+  return nextDayAsDate < dayAfterTomorrowAsDate || traverseWeekend || hasSlotsAfterSelectedDate(journal);
```

Comment:
Can we group the first logical operation in parentheses for clarity?
(nextDayAsDate < dayAfterTomorrowAsDate) || traverseWeekend || hasSlotsAfterSelectedDate(journal);